src/bindingHandlers/graphRenderer.ts

```diff
@@ -138,26 +139,56 @@ function render(graph: LogicalGraph, elementId : string, eagle : Eagle){
         .append("rect")
         .attr("class", "background");
 
-    const backgroundDragHandler = d3
-        .drag()
-        .on("start", function (node : Node) {
-            hasDraggedBackground = false;
-
-            if (d3.event.sourceEvent.shiftKey || d3.event.sourceEvent.altKey){
+    $("#logicalGraphD3Div svg").mousedown(function(e:any){
+        e.preventDefault()
+        hasDraggedBackground = false;
+        draggingInGraph = true;
+            if (e.shiftKey || e.altKey){
+                console.log("ah")
                 isDraggingSelectionRegion = true;
-                selectionRegionStart.x = d3.event.x;
-                selectionRegionStart.y = d3.event.y;
+                selectionRegionStart.x = DISPLAY_TO_REAL_POSITION_X(e.originalEvent.x);
+                selectionRegionStart.y = DISPLAY_TO_REAL_POSITION_Y(e.originalEvent.y-57.78-26);
```

Comment:
Can we make this number (57.78+26) a const, and put it near the top of the file.

I guess we should investigate determining this number at runtime too, in case we change the height of the navbar.